src/ansible/roles/facts/tasks/main.yml

```diff
@@ -1,6 +1,2 @@
-- name: 'Include distribution specific tasks [{{ ansible_distribution }} {{ ansible_distribution_major_version }}]'
-  include_tasks: '{{ include_file }}'
-  loop_control:
-    loop_var: include_file
-  with_first_found:
-  - files: '{{ ansible_distribution | distro_includes(ansible_distribution_major_version) }}'
+- name: 'Include distribution specific tasks [{{ ansible_distribution }}]'
```

Comment:
Please, keep the plugin. It may not be needed now, but it most certainly will be needed in the future.